frontend/src/components/plugins/forms/AnalyzerConfigForm.jsx

```diff
@@ -0,0 +1,612 @@
+                      key. ex: Authorization: &apos;Token &lt;api_key&gt;&apos;
+                    </small>
+                  </div>
+                </Col>
+              </Row>
+            </FormGroup>
+            <FormGroup>
+              <Row>
+                <Col md={2}>
+                  <Label className="me-2 mb-0" for="analyzer-api_key_name">
+                    Api key:
+                  </Label>
+                </Col>
+                <Col>
+                  <Input
+                    id="analyzer-api_key_name"
+                    type="text"
+                    name="api_key_name"
+                    value={formik.values.api_key_name}
+                    onBlur={formik.handleBlur}
+                    onChange={formik.handleChange}
+                    valid={
+                      !formik.errors.api_key_name && formik.touched.api_key_name
+                    }
+                    invalid={
+                      formik.errors.api_key_name && formik.touched.api_key_name
+                    }
+                    className="bg-darker border-0"
+                  />
+                  <div className="d-flex flex-column mt-1">
+                    <small className="mt-0 fst-italic">
+                      API key required for authentication. It will replace the
+                      &lt;api_key&gt; placeholder in the header.
+                    </small>
+                    {formik.touched.api_key_name &&
+                      formik.errors.api_key_name && (
+                        <small className="text-danger">
+                          {formik.errors.api_key_name}
+                        </small>
+                      )}
+                  </div>
+                </Col>
+              </Row>
+            </FormGroup>
+            <FormGroup>
+              <Row>
+                <Col md={2}>
+                  <Label className="me-2 mb-0" for="analyzer-certificate">
+                    Certificate:
+                  </Label>
+                </Col>
+                <Col>
+                  <Input
+                    id="analyzer-certificate"
+                    type="textarea"
+                    name="certificate"
+                    value={formik.values.certificate}
+                    onBlur={formik.handleBlur}
+                    onChange={formik.handleChange}
+                    valid={
+                      !formik.errors.certificate && formik.touched.certificate
+                    }
+                    invalid={
+                      formik.errors.certificate && formik.touched.certificate
+                    }
+                    className="bg-darker border-0"
+                    style={{
+                      minHeight: "100px",
+                      overflowX: "scroll",
+                    }}
+                  />
+                  <div className="d-flex flex-column mt-1">
+                    <small className="mt-0 fst-italic">
+                      Instance SSL certificate
```

Comment:
I'll modify this is something like: Add self signed SSL certificate for internal services